Core/src/Vertexing/GaussianTrackDensity.cpp

```diff
@@ -14,11 +14,12 @@
 
 namespace Acts {
 
-std::pair<double, double> Acts::GaussianTrackDensity::globalMaximumWithWidth(
+Result<std::optional<std::pair<double, double>>>
```

Comment:
This is a bit of an awkward type, but I don't see a better alternative either.

Reply:
agreed - we have this a couple of times now in the vertexing but I also don't see a way around